prow/config.yaml

```diff
@@ -6107,6 +6107,23 @@ periodics:
       - "--repo=k8s.io/autoscaler=master"
       - "--root=/go/src"
       - "--timeout=60"
+      env:
+      - name: DOCKER_IN_DOCKER_ENABLED
+        value: true
+      # docker-in-docker needs privileged mode
+      securityContext:
+        privileged: true
+      volumeMounts:
+      - name: docker-graph
+        mountPath: /docker-graph
+      # port set to limit to one job per node since sharing docker isn't safe
+      ports:
+      - containerPort: 9999
+        hostPort: 9999
+    volumes:
+    - name: docker-graph
+      hostPath:
+        path: /mnt/disks/ssd0/docker-graph
```

Comment:
you don't have a ssd mounted.. just use emptydir here

Reply:
Fixed.